website/source/guides/operations/plugin-backends.html.md

```diff
@@ -61,6 +61,14 @@ Put the plugin in the directory:
 $ mv my-mock-plugin /etc/vault/vault_plugins
 ```
 
+Alternatively, if you wanted a custom version of a plugin built into Vault, such as AppRole:
+
+```sh
+$ cd $GOPATH/src/github.com/hashicorp/vault/builtin/credential/approle/cmd/approle
```

Comment:
```suggestion
$ cd $GOPATH/src/github.com/hashicorp/vault/builtin/credential/approle/cmd
```

Reply:
Arg, I accepted this, but I now realize I shouldn't have. The original directory was correct: https://github.com/hashicorp/vault/tree/master/builtin/credential/approle/cmd/approle

I will push a commit including reverting that string.

Reply:
Oops, sorry about that. I happened to have `stable-website` checked out while running through the steps, and it's way old on code.